be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/RollCallHandler.scala

```diff
@@ -77,24 +77,16 @@ class RollCallHandler(dbRef: => AskableActorRef) extends MessageHandler {
               }
             }
 
-            val laoChannel: Option[Base64Data] = rpcRequest.getParamsChannel.decodeChannelLaoId
-            laoChannel match {
-              case None => Right(PipelineError(
-                ErrorCodes.SERVER_ERROR.id,
-                s"There is an issue with the data of the LAO",
-                rpcRequest.id
-              ))
-              case Some(_) =>
-                val combined = for {
-                  _ <- dbActor ? DbActor.ReadLaoData(rpcRequest.getParamsChannel)
-                  _ <- dbActor ? DbActor.WriteLaoData(rpcRequest.getParamsChannel, message)
-                } yield ()
-
-                Await.ready(combined, duration).value match {
-                  case Some(Success(_)) => createAttendeeChannels(data.attendees)
-                  case Some(Failure(ex: DbActorNAckException)) => Right(PipelineError(ex.code, s"handleCloseRollCall failed : ${ex.message}", rpcRequest.getId))
-                  case reply => Right(PipelineError(ErrorCodes.SERVER_ERROR.id, s"handleCloseRollCall failed : unexpected DbActor reply '$reply'", rpcRequest.getId))
-                }
+            //val laoChannel: Hash = rpcRequest.getParamsChannel.decodeChannelLaoId
+            val combined = for {
+              _ <- dbActor ? DbActor.ReadLaoData(rpcRequest.getParamsChannel)
```

Comment:
From what I understand, this is a good safety check, but if a message gets until a handler, doesn't it prove that the LAO already exists? (or does the ReadLaoData have another functionality here?)